src/Microsoft.PowerShell.Commands.Management/commands/management/GetComputerInfoCommand.cs

```diff
@@ -24,7 +26,6 @@ namespace Microsoft.PowerShell.Commands
     /// </summary>
     [Cmdlet(VerbsCommon.Get, "ComputerInfo",
         HelpUri = "https://go.microsoft.com/fwlink/?LinkId=799466")]
-    [Alias("gin")]
```

Comment:
For those aliases that are already declared in `InitialSessionState.cs`, we should keep them there for backward compatibility. For new aliases, I believe they should be declared using `AliasAttribute`, unless there is an explicit reason to set it with an option other than `None`.